langchain4j-core/src/main/java/dev/langchain4j/model/chat/request/json/JsonObjectSchema.java

```diff
@@ -241,6 +246,30 @@ public Builder addEnumProperty(String name, List<String> enumValues, String desc
             return this;
         }
 
+        /**
+         * Adds a single anyOf property to the properties of this JSON object.
+         * Please note that {@link #required(List)} should be set explicitly if you want the properties to be mandatory.
+         * @see #addAnyOfProperty(String, List, String)
+         * @see #addProperty(String, JsonSchemaElement)
+         * @see #properties(Map)
+         */
+        public Builder addAnyOfProperty(String name, List<JsonSchemaElement> anyOf) {
```

Comment:
We do not have convenience methods for complex types (objects, arrays), so please remove these new ones as well.
I am not sure they will be widely used, but they pollute the API quite a lot. We can always add them in the future if needed.

Reply:
Removed